apps/privacy-panel/js/ala/blur_slider.js

```diff
@@ -117,18 +117,18 @@ function() {
    */
   BlurSlider.getLabel = function(value) {
     switch(parseInt(value)) {
-      case 1:   return '500m';
-      case 2:   return '1km';
-      case 3:   return '2km';
-      case 4:   return '5km';
-      case 5:   return '10km';
-      case 6:   return '15km';
-      case 7:   return '20km';
-      case 8:   return '50km';
-      case 9:   return '75km';
-      case 10:  return '100km';
-      case 11:  return '500km';
-      case 12:  return '1000km';
+      case 1:   return '500 m';
+      case 2:   return '1 km';
+      case 3:   return '2 km';
+      case 4:   return '5 km';
+      case 5:   return '10 km';
+      case 6:   return '15 km';
+      case 7:   return '20 km';
+      case 8:   return '50 km';
+      case 9:   return '75 km';
+      case 10:  return '100 km';
+      case 11:  return '500 km';
+      case 12:  return '1000 km';
```

Comment:
Good point. I thought about it, and then forgot to check [CLDR for different behaviours](http://www.unicode.org/repos/cldr-aux/charts/26/by_type/units.length.html#9717dcedf60ac47).

I think it's safer to make them fully localizable. Example:
```
blur-unit-m = m
blur-unit-km = km
# LOCALIZATION NOTE (blur-distance): blur distance can vary from 500 m to 1000 km. 
# {{ unit }} will be replace with blur-unit-* strings.
blur-distance = {{ n }} {{ unit }}
```